src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(int side, String color) {

```

Comment:
Consider using the `Color` enum instead of a `String` for the color parameter to maintain type safety and consistency with the rest of the code.